src/steps/toolbx.rs

```diff
@@ -54,7 +57,7 @@ pub fn run_toolbx(ctx: &ExecutionContext) -> Result<()> {
             args.push("--yes");
         }
 
-        let _output = ctx.run_type().execute(&toolbx).args(&args).check_run();
```

Comment:
interesting that the output here was ignored

Reply:
Yeah, kinda weird. Maybe it has noisy output and they just wanted to hide it from the user?